polaris/hub/client.py

```diff
@@ -801,14 +823,53 @@ def _upload_benchmark(
                 "/v1/benchmark" if artifact_type == ArtifactSubtype.STANDARD.value else "/v2/competition"
             )
             url = f"{path_params}/{benchmark.owner}/{benchmark.name}"
-            response = self._base_request_to_hub(url=url, method="PUT", json=benchmark_json)
-            response_data = response.json()
+            self._base_request_to_hub(url=url, method="PUT", json=benchmark_json)
 
             progress_indicator.update_success_msg(
                 f"Your {artifact_type} benchmark has been successfully uploaded to the Hub. "
                 f"View it here: {urljoin(self.settings.hub_url, url)}"
             )
-            return response_data
+
+    def _upload_v2_benchmark(
+        self,
+        benchmark: BenchmarkV2Specification,
+        access: AccessType = "private",
+        owner: HubOwner | str | None = None,
+    ):
+        with ProgressIndicator(
+            start_msg="Uploading artifact...",
+            success_msg="Uploaded artifact.",
+            error_msg="Failed to upload benchmark.",
+        ) as progress_indicator:
+            # Get the serialized data-model
+            # We exclude the dataset as we expect it to exist on the hub already.
+            benchmark.owner = HubOwner.normalize(owner or benchmark.owner)
+            benchmark_json = benchmark.model_dump(exclude={"dataset"}, exclude_none=True, by_alias=True)
```

Comment:
**Comment:** Isn't it excluded by default?

```suggestion
            benchmark_json = benchmark.model_dump(exclude_none=True, by_alias=True)
```

Reply:
Good catch!